invenio_app_ils/mail/model.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (C) 2021 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Database model for emails."""
+
+import enum
+from datetime import datetime
+
+from celery.utils.log import get_task_logger
+from invenio_db import db
+
+celery_logger = get_task_logger(__name__)
+
+
+class TriggerEnum(enum.Enum):
+    """Enum for email trigger."""
+
+    MANUAL = "Manual"
+    AUTOMATICAL = "Automatical"
+
+
+class EmailLog(db.Model):
+    """Store the sent emails history."""
+
+    __tablename__ = "sent_emails"
+
+    id = db.Column(db.Integer, primary_key=True)
+
+    trigger = db.Column(db.String, nullable=False)
```

Comment:
Sorry... I got confused between the trigger and action. I propose to rename it. And why not use a [Boolean](https://docs.sqlalchemy.org/en/13/core/type_basics.html#sqlalchemy.types.Boolean) field?
```suggestion
    is_manually_triggered = db.Column(db.String, nullable=False)
```